### PR TITLE
add nlp-parse-from-file

### PR DESCRIPTION
**Pull request by AmeBel**
@williampma  please review

### CODE REVIEW

File: opencog/nlp/scm/processing-utils.scm [hunk 1]
```diff
@@ -338,9 +338,38 @@
...
+;
+(define (nlp-parse-from-file filepath)
+    (let*
+        ((cmd-string (string-join (list "cat " filepath) ""))
+        (port (open-input-pipe cmd-string))
+        (line (get-line port))
+        )
+        (while (not (eof-object? line))
+            (if (or (= (string-length line) 0)
+                    (char=? #\; (string-ref (string-trim line) 0))
```
Comment (williampma) [inline opencog/nlp/scm/processing-utils.scm]: So the file is a bunch of sentences, and ";" comment out one line?

Comment (AmeBel) [inline opencog/nlp/scm/processing-utils.scm]: yes

Comment (AmeBel) [inline opencog/nlp/scm/processing-utils.scm]: i was thinking scheme when i choose that :-) maybe # is better/popular ???

Comment (williampma) [inline opencog/nlp/scm/processing-utils.scm]: I think `;` is fine :)

**Comment by AmeBel** [comment]
Thanks

**Pull request merged**